### DISCUSSION TITLE
How people is tagged for other users?

### DISCUSSION THREAD

**Discussion by DavidMndz** [nextcloud/recognize#704] [Q&A] [answered]
I have a test Nextcloud instance where I have this software running and so far, it works pretty well.

I am testing by adding new users to Nextcloud and sharing some images with them, but it is not fully clear to me how the faces are managed in this way. Are the pictures re-scanned per user creating new clusters? Or how does this work?

About the tags for the images, I already have seen that at Nextcloud level all the tags are shown but in the Photos app only those with images shared appear. 

Thanks!

**Comment by marcelklehr** [comment, marked as answer]
> it is not fully clear to me how the faces are managed in this way. Are the pictures re-scanned per user creating new clusters? Or how does this work?

Currently, pictures are rescanned per user and every user has their own clusters.

> I already have seen that at Nextcloud level all the tags are shown but in the Photos app only those with images shared appear.

Yes, that's correct. For convenience the photos app only shows tags that you have photos for. Generally in Nextcloud you can see all tags anyone has ever created, though.

**Comment by DavidMndz** [comment]
Thanks a lot for clarifying, this was exactly what I was looking for.

May I suggest add this information into the wiki? I missed this kind of details and also about how everything is stored in the DB, just in case some admins want to remove and cleanup everything.

Thanks again.
